src/Inventory.js

```diff
@@ -6,17 +6,15 @@ class Inventory extends Map {
    */
   addItem(item) {
     this.set(item.uuid, item);
-    item.isHeld = true;
-    item.container = this;
+    item.belongsTo = this;
```

Comment:
This is still wrong, heh. `belongsTo` should be equal to the _entity_ (either Character or Item) it belongs to. Not the actual Inventory instance. Otherwise you just kick the can down the road because sure you have a pointer to the inventory it belongs to but you can't actually get a reference to entity that the inventory belongs to. So you could never, for example, get the name of the player that is carrying an item simply by having the item.